### PR TITLE
Fixed #383

### LINKED ISSUE #383: Show icon of speakers
## Overview (Required)

Now icon images of speakers are available from speakers.json. 😉

## Links

- web: https://droidkaigi.github.io/2017/timetable.html#session-39
- JSON: https://droidkaigi.github.io/2017/sessions.json

### PR DESCRIPTION
**Pull request by henteko**
## Issue
- #383 

## Overview (Required)
- Show icon of speakers 🎉

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by KeithYokoma** [comment]
LGTM!!!

**Comment by konifar** [comment]
🆒 !!! Thanks for quick work! 👍 
LGTM! 💯

**Comment by konifar** [comment]
@KeithYokoma Also thank you so much for quick review! You must have "two-sword" extra skill.

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/DataBindingHelper.java [hunk 1]
```diff
@@ -78,6 +78,14 @@ private static void setImageUrlWithSize(ImageView imageView, @Nullable String im
         }
     }
 
+    private static String convertToFullImageUrl(String imageUrl) {
+        if (imageUrl.startsWith("/")) {
+            return "https://droidkaigi.github.io/2017" + imageUrl;
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/DataBindingHelper.java]: Ah, sorry, I think this should be converted in ViewModel.
Can you refactor on another PR? 🙇

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/DataBindingHelper.java]: Okay, let me try 💪

**Comment by henteko** [comment]
Thank you!!!!!!!!!

**Comment by konifar** [comment]
@henteko Sorry 🙇 I refactored this fix. Can you take a look?
https://github.com/DroidKaigi/conference-app-2017/pull/386

**Pull request merged**